src/main.go

```diff
@@ -24,30 +26,26 @@ func main() {
 	// Channel for sending BGP messages between parsing and processing
 	msgChannel := make(chan common.BGPMessage)
 
-	// Channel for sending windows from processing to analyzing
-	windowChannel := make(chan common.Window)
-
-	wg.Add(3)
+	wg.Add(2)
```

Comment:
this is a minor nit, but: you could do `wg.Add(1)` next to each of the two remaining worker calls, which then makes things clear next time that has to be edited